docs/shared/recipes/nx-release/publish-rust-crates.md

```diff
@@ -0,0 +1,216 @@
+# Using Nx Release with Rust
+
+This recipe guides you through versioning Rust libraries, generating changelogs, and publishing Rust crates in a monorepo with Nx Release.
+
+{% github-repository url="https://github.com/JamesHenry/release-js-and-rust" /%}
```

Comment:
We should probably put this repository in the `nrwl` organization

Reply:
I will do that in a follow up, don't want to lose admin control just yet